material-sheet-fab/build.gradle

```diff
@@ -29,7 +29,7 @@ android {
     buildToolsVersion "23.0.3"
 
     defaultConfig {
-        minSdkVersion 14
+        minSdkVersion 15
```

Comment:
Ah I see. Their readme still says "Lollipop ViewAnimationUtils.createCircularReveal for everyone 2.3+", but the release notes mention the change in SDK support. In that case, I would rather use an older version of CircularReveal than change the SDK support of this library.